prboom2/src/r_draw.c

```diff
@@ -479,9 +480,11 @@ void R_InitBuffer(int width, int height)
 
   drawvars.topleft = screens[0].data;
   drawvars.pitch = screens[0].pitch;
-
-  for (i=0; i<FUZZTABLE; i++)
```

Comment:
We don't need declaration of `i` variable if you remove this cycle.

Cycle is not a bad idea, maybe we should implement it in Woof as well.

Reply:
Reimplemented cycle, even though it just saves 1 mult operation